.eslintcache

```diff
@@ -0,0 +1 @@
+[{"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\index.tsx":"1","C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\reportWebVitals.ts":"2","C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\App.tsx":"3","C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Calendar.tsx":"4","C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Header.tsx":"5","C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Params.tsx":"6"},{"size":517,"mtime":1611947373932,"results":"7","hashOfConfig":"8"},{"size":440,"mtime":1611932766260,"results":"9","hashOfConfig":"8"},{"size":1035,"mtime":1612111961172,"results":"10","hashOfConfig":"8"},{"size":244,"mtime":1612198942594,"results":"11","hashOfConfig":"8"},{"size":616,"mtime":1612198962134,"results":"12","hashOfConfig":"8"},{"size":2426,"mtime":1612199353379,"results":"13","hashOfConfig":"8"},{"filePath":"14","messages":"15","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},"1vhnj7t",{"filePath":"17","messages":"18","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"19","messages":"20","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"16"},{"filePath":"21","messages":"22","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"23","messages":"24","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"25","messages":"26","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\index.tsx",[],["27","28"],"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\reportWebVitals.ts",[],"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\App.tsx",[],"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Calendar.tsx",[],"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Header.tsx",[],"C:\\Users\\Patryczegusio\\_dev\\OrangeWorkSchedule-front\\src\\Params.tsx",[],{"ruleId":"29","replacedBy":"30"},{"ruleId":"31","replacedBy":"32"},"no-native-reassign",["33"],"no-negated-in-lhs",["34"],"no-global-assign","no-unsafe-negation"]
```

Comment:
No need to add .eslintcache to the commit

Reply:
done